package/yast2-network.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Dec 23 09:21:16 UTC 2020 - Knut Anderssen <kanderssen@suse.com>
+
+- Improve additional addresses validation (bsc#1174766)
+- 4.3.37
```

Comment:
In the previous SR, the version was already increased to 4.3.37 by error.

see https://github.com/yast/yast-network/pull/1134/files#diff-a07b34dd719aba232c4133a770816f1c6c41f8a44bf4347910afbd0d943d16acR20